migrate/labels.yml

```diff
@@ -0,0 +1,118 @@
+# Every item must have a name, color, and description.
+
+# Notes:
+# * Repositories may define their own labels as long as they don't
+#   conflict with what's listed here. If it turns out a repository-specific
+#   label would be useful to have in the entire organization, we can always add
+#   it here post facto.
+# * We have added these based on need. They are not all-encompassing. If something
+#   seems missing, feel free to open a PR.
+# * Whenever a repo has a matching label, that label will be updated. If there
+#   is no matching label, the label will be created anew. "Matching" is determined
+#   by comparing the label's lowercased name, with all puncutation, emoji, and special
+#   chars removed.
+# * For picking hex colors, you might use: https://www.htmlcolor-picker.com/
+#   Describe your color with a comment so it's easier to review.
+
+
+### LABELS INDICATING HOW ISSUES CAN BE ENAGAGED WITH.
+
+- name: "good first issue"
+  color: "42dd35"  # friendly lime green
+  description: "A good task for a newcomer to start with"
+
+- name: "help wanted"
+  color: "54976d"  # fenway green
+  description: "Ready to be picked up by anyone in the community"
+
+
+### LABELS INDICATING BROAD THEMES OF WORK.
+### MORE THAN ONE OF THESE MAY APPLY AT A TIME.
+
+- name: "maintenance"
+  color: "ff9125"  # construction orange
+  description: "Relates to platform health and/or routine upkeep"
+
+- name: "security"
+  color: "f0d165"  # caution yellow
+  description: "Relates to improving to the security posture of the platform"
+
+- name: "enhancement"
+  color: "a2eeef"  # electric blue!
+  description: "Relates to new features or improvements to existing features"
+
+- name: "documentation"
+  color: "0052cc"  # royal blue
+  description: "Relates to documentation improvements"
+
+
+### LABELS INDICATING THE SCOPE OR FUNCTION OF THE ISSUE.
+### AT MOST ONE OF THESE SHOULD BE USED AT A TIME.
+
+- name: "depr"
+  color: "3c3cc3"  # indigo-blue
+  description: "Proposal for deprecation & removal per OEP-21"
+
+- name: "bug"
+  color: "d93f0b"  # scarlet red...
+  description: "Report of or fix for something that isn't working as intended"
+
+- name: "discovery"
+  color: "d876e3"  # a curious shade of lavendar
+  description: "Pre-work to determine if an idea is feasible"
+
+- name: "epic"
+  color: "7e027e"  # epic violet
+  description: "Large unit of work, consisting of multiple tasks"
+
+- name: "initiative"
+  color: "340963"  # formidably deep mauve
+  description: "Huge unit of work, consisting of multiple epics"
+
+
+### LABELS INDICATING ISSUE OUTCOMES.
+
+- name: "wontfix"
+  color: "ffffff"  # white
+  description: "This will not be worked on"
+
+- name: "duplicate"
+  color: "cfd3d7"  # duplicitous grey
+  description: "This issue or pull request already exists elsewhere"
+
+
+### LABELS USED IN THE OPEN EDX CONTRIBUTION WORKFLOW.
+### PLEASE CONSULT WITH OPEN EDX PROJECT MANAGMENT BEFORE CHANGING THESE.
```

Comment:
@kdmccormick I added a [section](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3548807177/Community+Contributions+Project+Manager#Current-OSPR-Project-Managers) listing current holders of the OSPR Project Manager role to the wiki page on OSPR management.

Perhaps it would be helpful to link to that here, so whoever reads this can easily look up who to ping?

CC @mphilbrick211

Reply:
Perfect, I'll add that in.